services/blockchain-indexer/shared/indexer/transactionProcessor/token/transfer.js

```diff
@@ -49,7 +50,7 @@ const applyTransaction = async (blockHeader, tx, dbTrx) => {
 
 	tx.amount = tx.params.amount;
 	tx.data = tx.params.data;
-	tx.recipientAddress = tx.params.recipientAddress;
+	tx.recipientAddress = getLisk32Address(tx.params.recipientAddress);
```

Comment:
Please move this change to the connector instead within the `decodeTransaction` method.